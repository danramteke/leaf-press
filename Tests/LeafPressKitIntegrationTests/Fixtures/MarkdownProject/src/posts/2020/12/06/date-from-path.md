---
title: Date from Path
---
#  Date from path example

LeafPress can determine the date of a post from its path.  

Static resources are copied over verbatim so that images that go with the blog post are in a nearby folder.

![image](images/image.png)

Here is an image in the `images` sub-folder.

