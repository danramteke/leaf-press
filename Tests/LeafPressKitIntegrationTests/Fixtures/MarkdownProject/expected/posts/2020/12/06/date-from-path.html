<html>
  <head>
    <link rel="stylesheet" href="/styles.css">
  </head>
  <body>
    <div id="post-content">


<h1>Date from path example</h1>
<p>LeafPress can determine the date of a post from its path.</p>
<p>Static resources are copied over verbatim so that images that go with the blog post are in a nearby folder.</p>
<p><img src="images/image.png" alt="image" /></p>
<p>Here is an image in the <code>images</code> sub-folder.</p>



    </div>
  </body>
</html>
